user/urls.py

```diff
@@ -1,10 +1,17 @@
 from django.urls import path
-from user.views import CreateUserView, CreateTokenView, ManageUserView
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+    TokenVerifyView,
+)
+from user.views import CreateUserView, ManageUserView
```

Comment:
Add one blank line between the different groups of imports.

Reply:
Thanks!